tests/test_maneuver_guidance.py

```
@@ -0,0 +1,577 @@
+        self.assertTrue(all(item.citation == self.spec.source for item in narrative_items))
+
+
+class _ConstantAssumptionResponseModel:
+    name = "synthetic test response — not SR22 performance"
+
+    def resolve(self, state, flight_input, environment) -> QuasiSteadyResponse:
+        """
+        Provide a fixed synthetic quasi-steady aircraft response for testing.
+        
+        Returns:
+            QuasiSteadyResponse: A response representing 110 kt true airspeed, a −2° flight-path angle, and an assumed model result.
```

Ruff (0.16.1)

[warning] Docstring contains ambiguous `−` (MINUS SIGN). Did you mean `-` (HYPHEN-MINUS)?

(RUF002)